packages/agent/src/polling/index.ts

```diff
@@ -14,6 +14,9 @@ export type PollStrategy = (
 ) => Promise<void>;
 export type PollStrategyFactory = () => PollStrategy;
 
+// Default delta for ingress expiry is 5 minutes.
+const DEFAULT_INGRESS_EXPIRY_DELTA_IN_MSECS = 5 * 60 * 1000;
```

Comment:
The rust agent seems to use 3min.  https://github.com/dfinity/agent-rs/blob/main/ic-agent/src/agent/agent_config.rs#L43

Do we wanna use 3min as the default in the agent-js as well?

Reply:
Btw, a constant with the same name and the same value already exists in agent/http/index.ts
https://github.com/dfinity/agent-js/pull/938/files#diff-1deee5c6f68eb82e2be634c32d050e818f857292ed0fa9c9e19026e2ef60869cR57